tests/manager.rs

```diff
@@ -134,3 +138,120 @@ fn test_same_with_capacity_safe() {
     let fetched_arc = manager.get(p).expect("success").expect("existed");
     assert!(Arc::ptr_eq(&created_arc, &fetched_arc));
 }
+
+/// Some storage drivers are able to recover when the database is corrupted at runtime.
+/// Test how these managers can discard corrupted databases and re-open.
```

Comment:
Nit: this test is to verify that a corrupted database can be opened with `discard_if_corrupted`, but the comment (which is exactly the same as the one of the next test) sounds more about being able to recover from the db corruption, which is a bit misleading IMO.

Reply:
Reworded.